MIGRATION.md

```diff
@@ -2,6 +2,35 @@
 
 When new releases include breaking changes or deprecations, this document describes how to migrate.
 
+## Migrating to 1.11.0
+
+### Breaking changes
+
+`FreshnessPolicy` (deprecated as of `1.6`) has been renamed to `LegacyFreshnessPolicy`. The deprecated freshness policies will henceforth be referred to in docs and code as "legacy freshness policies". No changes in functionality.
+
+You can still import and use the legacy freshness policies from the `deprecated` module:
+
+```
+from dagster.deprecated import FreshnessPolicy  # imports LegacyFreshnessPolicy
+```
+
+Existing imports of `FreshnessPolicy` will fail with an `ImportError`:
```

Comment:
I would swap the order of this info and the previous block (tell people that the existing import will break, then tell them how to import them if they really need to)